docs/processing_information.md

```diff
@@ -95,18 +96,23 @@ As a consequence, cells which `CellAssign` cannot confidently annotate from the
 Please be aware that all cell type annotation reference datasets are derived from normal (not tumor) tissue.
 In addition, `CellAssign` annotation is only performed if there are at least 30 cells present in the `processed` object.
 
-Some cells may be labeled as "Unclassified cell" if they were not annotated with `SingleR` or `CellAssign`.
+For `SCimilarity` annotation, we use the foundational model described in [Heimberg _et al._ 2025](https://doi.org/10.1038/s41586-024-08411-y) that contains 7.3 million cells from various normal and diseased tissues to annotate all samples.
+Cells will be annotated with the cell type label of the cell from the model that is most similar to the query cell being annotated. 
+
+Some cells may be labeled as "Unclassified cell" if they were not annotated with `SingleR`, `CellAssign`, or `SCimilarity`.
 These are cells which were not present in previous ScPCA data versions on which cell typing was initially performed, so they were not labeled.
 
-Additionally, annotations from `SingleR` and `CellAssign` are used to assign an ontology-aware consensus cell type label.
-The [latest common ancestor (LCA)](https://rdrr.io/bioc/ontoProc/man/findCommonAncestors.html) between the `SingleR` and `CellAssign` cell type assignments is used as the consensus cell type label if the following criteria are met, otherwise no consensus cell type is assigned:
+Additionally, annotations from `SingleR`, `CellAssign`, and `SCimilarity` are used to assign an ontology-aware consensus cell type label.
```

Comment:
Something I wonder for docs in general - is there room for a subsection within cell type annotation for "Consensus cell types"? This would be helpful as a relative link in a handful of spots.